api/permissions.py

```diff
@@ -13,6 +13,6 @@ def has_object_permission(self, request, view, obj):
         return False
 
 
-class IsProfileOwnerOrStaff(IsAdminOrSelf):
+class IsPlaceOwnerOrAdmin(IsAdminOrSelf):
```

Comment:
Why the name change on the class? Shouldn't it be a bit more generic?

And instead of returning obj.user, if it is the *owner* of the object, shouldn't that be an `.owner` attribute? (So that the very attribute name describes the relationship purpose)